public/cli/lib/commands/logger.ts

```diff
@@ -0,0 +1,83 @@
+import { join } from 'path';
+import fs from 'fs-extra';
+import { format as dateFormat } from 'date-fns';
+import winston from 'winston';
+import os from 'os';
+
+// Importing isDev causes an error running tests as we don't run in an
+// electron environment. Ava sets the NODE_ENV to 'test'.
+let isDev = false;
+if (process.env.NODE_ENV !== 'test') {
+  isDev = require('electron-is-dev');
+}
+// Sets up the logger. Should be called when opening the app.
+export function prepareLogger(numLogsToKeep = 10, pathToLogDir = '') {
+  // make sure the logging directory exists
+  const homedir = os.homedir();
+  if (pathToLogDir == '') {
+    switch (process.platform) {
+      case 'darwin':
+        pathToLogDir = join(homedir, 'Library', 'Logs', 'bible-karaoke');
+        break;
+
+      case 'win32':
+        pathToLogDir = join(homedir, 'AppData', 'Roaming', 'bible-karaoke', 'logs');
+        break;
+
+      default:
+        pathToLogDir = join(homedir, '.config', 'bible-karaoke', 'logs');
+        break;
+    }
+  }
+  fs.mkdirsSync(pathToLogDir);
+
+  // remove log files that are > numLogsToKeep
+  const entries = fs.readdirSync(pathToLogDir);
+  while (entries && entries.length > numLogsToKeep) {
+    const fileToRemove = entries.shift();
+    if (fileToRemove != undefined) {
```

Comment:
Use `!==` for comparison (except where you want coercion). The only place to use coercion is `!= null` or `== null` which neatly checks for both null and undefined. Perhaps that's what you want here?